server/app/Icepeak/Main.hs

```diff
@@ -52,20 +53,20 @@ main = do
   config <- execParser (configInfo env)
 
   -- start logging as early as possible
-  logger <- Logger.newLogger (fromIntegral $ configQueueCapacity config)
+  logger <- Logger.newLogger config
   loggerThread <- Async.async $ Logger.processLogRecords logger
+  SentryLogging.runWithCrashLogger "Control loop error" (Logger.loggerSentryService logger) $ do
```

Comment:
All errors caught by this would now go only via sentry and not be logged wherever else.
This handler should probably also just post the log message to the logger.